### ISSUE TITLE
Badge is drawn under buffer name instead of next to it

### ISSUE THREAD

**Issue by Vaesper** [glowing-bear/glowing-bear#863]
With long buffer names it looks like this:

![image goes here](https://i.imgur.com/RJlQzzs.png)

**Comment by lorenzhs** [comment]
Yup, the clipping only prevents overflow beyond the bufferlist, not the badges. Probably would be easiest to fix by having the badge affect the name label's size
